posthog/api/event.py

```diff
@@ -100,7 +100,7 @@ class EventViewSet(StructuredViewSetMixin, mixins.RetrieveModelMixin, mixins.Lis
 
     def get_queryset(self):
         queryset = cast(EventManager, super().get_queryset()).add_person_id(self.team_id)
-        if self.action == "list" or self.action == "sessions" or self.action == "actions":
```

Comment:
Not sure about this one

Reply:
I don't think it's used anywhere, this action comes from the URL definition or DRF's default behavior which does not set a `sessions` action (nor an `actions` action for that matter). Tests pass too so I think it's safe to merge.